lib/draft/feature_test_phoenix_integration.md

```diff
@@ -0,0 +1,232 @@
+# Using Feature Tests to Maximize Business Value
+
+Keeping focus on providing value for the business is a crucial part in making sure stakeholders have confidence in you and your development team. In other words, if you want to keep your job, you better provide some value! The flipside of this is that I want to keep my development team happy so they'll keep on writing quality code they are proud of. Let's find out how to use our favorite tools (Elixir & Phoenix) and see how to make sure we deliver the value our stakeholders expect...
+
+## User Story
+
+I am a big proponent of using the scrum process to deliver software that adds value in small iterations. A tool that is used in this process is the concept of a user story. Writing a good user story should reveal what the business wants and should also indicate why the business needs certain functionality.
+
+The form of a user story is mostly this: _As a [user of the application] I would like to [do a certain thing (mostly in the application)] in order to [provide certain value or solve a certain problem]_
+Writing good and clear user stories is not a trivial task and once you have a good user story the next step might even be harder... Translating that user story into the code you are going to write as a developer. Luckily there are already great concepts and methods in the software industry that try to bridge that gap. One such process is the process of BDD (Behaviour Driven Development). Which basically means we build features from the outside In, meaning we start with a high level of what the app needs to do and granually start to fill in more details as we are building the feature.
+
+## Acceptance Criteria using Gherkin
+
+The way that I like to tackle the process of translating a user story into managable chunks to build is by defining a good set of acceptance criteria. It is often a good idea to write these with a product owner or stakeholder from the business, as they are the ones that are going to be accepting the story in the end. It is a feature that they have requested, right? I really like the [Gherkin language](https://en.wikipedia.org/wiki/Cucumber_(software)#Gherkin_language) to write up my acceptance criteria. If you have used the Cucumber testing framework this will look very familiar.
```

Comment:
>  If you have used the Cucumber testing framework this will look very familiar.
Cucumber is mostly used in Rails Land isn't it? Maybe mention that here, since the reader might come from a different domain.

>  If you have used the Cucumber testing framework in Rails land, this will look very familiar.

Just an idea